Swift/ChartboostExample/ViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  ViewController.swift
+//  ChartboostExample
+//
+//  Created by Arnau Marti on 08/07/2020.
+//  Copyright © 2020 Chartboost. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, CHBInterstitialDelegate, CHBRewardedDelegate, CHBBannerDelegate {
+
+    private lazy var interstitial = CHBInterstitial(location: CBLocationDefault, delegate: self)
+    private lazy var rewarded = CHBRewarded(location: CBLocationDefault, delegate: self)
+    private lazy var banner = CHBBanner(size: CHBBannerSizeStandard, location: CBLocationDefault, delegate: self)
+    private var logBeforeViewDidLoad = String()
+
+    @IBOutlet weak var textView: UITextView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        log(message: logBeforeViewDidLoad)
+    }
+
+    //MARK: - IBActions
+
+    @IBAction func cacheInterstitial(_ sender: Any) {
+        interstitial.cache()
+    }
+
+    @IBAction func showInterstitial(_ sender: Any) {
+        // If the interstitial is not cached didShowAd:error: will be called with an error.
+        interstitial.show(from: self)
+    }
+
+    @IBAction func cacheRewarded(_ sender: Any) {
+        rewarded.cache()
+    }
+
+    @IBAction func showRewarded(_ sender: Any) {
+        // We can let showFromViewController: fail for not-cached ads as we do in showInterstitial:, or preemtively check against the isCached property before calling it:
+        if rewarded.isCached == true {
```

Comment:
I think `if rewarded.isCached {` is better. This was probably necessary when rewarded was an optional but not anymore.